package.json

```diff
@@ -44,18 +23,11 @@
     "bignumber.js": "5.0.0"
   },
   "devDependencies": {
-    "babel": "~5.8.23",
-    "chai": "~3.2.0",
-    "mocha": "~2.3.0",
-    "onchange": "~2.0.0",
-    "standard": "~5.2.1"
+    "standard": "^17.1.0",
+    "vitest": "^0.34.2"
   },
   "scripts": {
-    "build": "babel src -d dist",
```

Comment:
The ["Contributing" section of the README](https://github.com/vend/vend-number#contributing) still refers to some these scripts; can we update?